presto-docs/src/main/sphinx/release/release-0.232.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.232
+=============
+
+General Changes
+_______________
+* Fix an issue where ``DATE_TRUNC`` may produce incorrect results at certain timestamp in ``America/Sao_Paulo``.
+* Improve built-in function resolution performance by caching function resolution results.
+* Add a MySQL-based function namespace manager implementation that supports creating, altering, and dropping SQL functions (:doc:`/admin/function-namespace-managers`).
+* Add support for retrying failed stages from a materialized point instead of failing the entire query. The number of retries allowed can be configured using the configuration property ``max-stage-retries`` and session property ``max_stage_retries``. The default value is zero. To take advantage of this feature, exchange_materialization_strategy must be set to ``ALL``.
+* Add configuration property ``use-legacy-scheduler`` and session property ``use_legacy_scheduler`` to use a version of the query scheduler before refactorings to enable full stage retries. The default value is false. This is a temporary property to provide an easy way to roll back in case of bugs in the new scheduler. This property will be removed in a couple releases once we have confidence in the stability of the new scheduler.
+* Add ``query_max_total_memory_per_node`` and ``query_max_memory_per_node session`` properties.
+* Add support to show whether functions have variable arity in ``SHOW FUNCTIONS``.
+* Add support to show whether functions are built-in in ``SHOW FUNCTIONS``.
+* Add ``use_exact_partitioning`` session property that forces repartitioning if repartitioning is possible.
+* Add support for ``ALTER FUNCTION``.
+* Add configuration property ``resource-groups.reload-refresh-interval`` to control the frequency of reloading resource group information from the database. The default value is 10 seconds.
+* Add config ``internal-communication.task-communication-protocol`` to support Presto nodes to shuffle data with Thrift protocol. Use HTTP or Thrift.
+* Add config ``internal-communication.server-info-communication-protocol`` to support Presto nodes to announce state with Thrift protocol. Use HTTP or Thrift.
```

Comment:
Add support for using the Thrift protocol to announce node state. This can be configured using the configuration property  ``internal-communication.server-info-communication-protocol``.  Possible values are HTTP or Thrift.